addon/model.js

```diff
@@ -50,7 +50,7 @@ var attr = function() {
                 this.set("_oldState", clone(this));
             }
 
-            var ready = (value === "" || value === undefined) && (data[key] === undefined);
+            var ready = (value === "" || Ember.isNone(value)) && (Ember.isNone(data[key]));
```

Comment:
this line wasn't under test so I'll add something to show this is needed/used/works/or doesn't work